snaketalk/function.py

```diff
@@ -238,7 +238,7 @@ def ensure_response(*args):
         try:
             self.function(self.plugin, event)
         except Exception:
-            logging.error(exc_info=True)
+            logging.exception("Exception occurred: ")
         finally:
```

Comment:
The `exc_info` was supposed to print the traceback here (but I didn't test it so it was probably broken). This change doesn't print any details about the error at all?